speech/speech_samples.rb

```diff
@@ -25,9 +25,11 @@ def speech_sync_recognize project_id: nil, audio_file_path: nil
                                          language:    "en-US"
 
   results = audio.recognize
-  result  = results.first
 
-  puts "Transcription: #{result.transcript}"
+  # Each result represents a consecutive portion of the audio
```

Comment:
Remove this comment for each change in `speech_samples.rb`

The quickstart is heavily commented but not the documentation snippets that come from this file

Reply:
Done.